### PR TITLE
Ch10155/callstats android library integration in apprtc android demo app

### PR DESCRIPTION
**Pull request by pallab-gain**
https://app.clubhouse.io/callstatsio/story/10155/callstats-android-library-integration-in-apprtc-android-demo-app

**Callstats android library integration in AppRTC android demo app**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,52 @@
-# AppRTCMobile_android
+# AppRTC Android demo 
+
+AppRTC demo application with Callstats.io android library integration
+
+## Getting Started
+
+These instructions will help you run the apprtc android demo application with csio integration. See notes on how you will get
```
Comment (karthikbr82) [inline README.md]: change csio to callstats.io

File: README.md [hunk 2]
```diff
@@ -1 +1,52 @@
...
+callstarts library related credentials to run the application.
```
Comment (karthikbr82) [inline README.md]: callstarts to callstats

File: README.md [hunk 3]
```diff
@@ -1 +1,52 @@
...
+    - [ x ] [ICE events ](https://docs.callstats.io/restapi/#tag/ICE-Events)
+        - [ x ] ICE connection, gathering, signaling change
+    - [ x ] [Media Events](https://docs.callstats.io/restapi/#tag/Media-Events)
+        - [ x ] Mutes/unmutes
+        - [ x ] Video play/pause
+    - [ x ] [Special Event - application log ](https://docs.callstats.io/restapi/#tag/special-events)
+        - [ x ] Application specific log. Can be anything that you find should be important for your application
+        - [ x ] Example - Toggle between front/rear camera, when user connected/disconnected/join/leave apprtc room, etc.
+    
+
```
Comment (karthikbr82) [inline README.md:183]: can you write some description about the integration steps?

File: app/src/main/java/org/appspot/apprtc/csio/CsioSHIM.java [hunk 4]
```diff
@@ -1,8 +1,216 @@
...
+    }
+
+    @Subscribe
+    public void OnLogs(CSIOEvents.OnLogs onLogs) {
+        Logging.d(TAG, "OnLogs ");
+        if (callstats != null) {
+            callstats.log(onLogs.message, onLogs.level, LoggingType.TEXT);
+        }
+    }
+
```
Comment (karthikbr82) [inline app/src/main/java/org/appspot/apprtc/csio/CsioSHIM.java:215]: can we use camel case for naming?

**Review by karthikbr82 — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (pallab-gain) [reply README.md:183]: Thanks. I just added integration steps, and pushed for your review.

**Comment by pallab-gain** [comment]
@karthikbr82 I updated the readme. Can you please take a look, and let me know if we need some updates or fix ?

- Add integration steps
- Some other readme, and code fix based on your previous feedback.

File: app/src/main/java/org/appspot/apprtc/csio/CSIOEvents.java [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+        String message;
+        LoggingLevel level;
+
+        public OnLogs(String _message, LoggingLevel _level) {
+            message = _message;
+            level = _level;
+        }
+    }
+
+}
```
Comment (karthikbr82) [inline app/src/main/java/org/appspot/apprtc/csio/CSIOEvents.java:93]: use camel case for function naming.

**Review by karthikbr82 — APPROVED** [review]

**Pull request merged**